backend/zeno_backend/server.py

```diff
@@ -92,6 +82,7 @@ def get_server() -> FastAPI:
     api_app = FastAPI(
         title="Backend API", generate_unique_id_function=lambda route: route.name
     )
+    api_app.include_router(sdk.router)
```

Comment:
Nice one, wanted to split up the two more as well. However, do we want to also differentiate the parent paths? Like we have `api` rn, do we want to add a separate called `cli` or something? Also, do we want these to have a tag as well. Like we have the 'zeno' tag do we want to add another one?

Reply:
hmm interesting point, maybe we make it /sdk or something?

Reply:
Will make a Linear issue :)